tests/ttnn/unit_tests/operations/test_reduction.py

```diff
@@ -48,3 +48,25 @@ def test_var(device, batch_size, h, w, dim):
 
     output_tensor = ttnn.to_torch(output_tensor)
     assert_with_pcc(torch_output_tensor, output_tensor, pcc=0.99)
+
+
+@pytest.mark.parametrize("batch_size", [1, 16, 1, 16])
```

Comment:
Please don't have duplicates. Just `[1,16]`

Reply:
Can be left for next PR.

Reply:
ohh yes. its typo. Will fix it